src/slack/slack.js

```diff
@@ -1144,3 +1142,15 @@ async function orderInfo(items, country) {
   }
   return {info: {items: info, totalPrice, country: orderCountry, wrongCountry}}
 }
+
+function userOrderAttachment(callback_id, actions, fields, pretext, title) {
+  return {
+    pretext,
+    fields,
+    actions,
+    callback_id,
+    fallback: ' ',
```

Comment:
please take the moment to update this to something reasonable as well - I think the `pretext` would make sense to be passed here, but check it out in docs and test it